packages/mobx-react-lite/__tests__/observer.test.tsx

```diff
@@ -997,38 +997,6 @@ it("dependencies should not become temporarily unobserved", async () => {
     expect(doubleDisposed).toBeCalledTimes(1)
 })
 
-it("Legacy context support", () => {
```

Comment:
legacy context is gone in R19

Reply:
I think it is fine to disable that that test (`it.skip("...`). In the next major we're going to drop it anyway

Reply:
skipping instead